.env

```
@@ -90,3 +90,15 @@ MGATE_HTTP_WITH_MTLS_SERVER_CA_FILE=ssl/certs/ca.crt
 MGATE_HTTP_WITH_MTLS_CLIENT_CA_FILE=ssl/certs/ca.crt
 MGATE_HTTP_WITH_MTLS_CERT_VERIFICATION_METHODS=ocsp
 MGATE_HTTP_WITH_MTLS_OCSP_RESPONDER_URL=http://localhost:8080/ocsp
+
+MPROXY_COAP_WITHOUT_DTLS_ADDRESS=:5682
+MPROXY_COAP_WITHOUT_DTLS_TARGET=localhost:5683
+
+MPROXY_COAP_WITH_DTLS_ADDRESS=:5684
+MPROXY_COAP_WITH_DTLS_TARGET=localhost:5683
+MPROXY_COAP_WITH_DTLS_CERT_FILE=ssl/certs/server.crt
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MPROXY_COAP_WITH_DTLS_CERT_FILE key should go before the MPROXY_COAP_WITH_DTLS_TARGET key

(UnorderedKey)